lib/active_model_serializers/lookup_chain.rb

```diff
@@ -0,0 +1,70 @@
+module ActiveModelSerializers
+  module LookupChain
+    # Standard appending of Serializer to the resource name.
+    #
+    # Example:
+    #   Author => AuthorSerializer
+    BY_RESOURCE = lambda do |resource_class, _serializer_class, _namespace|
+      serializer_from(resource_class)
+    end
```

Comment:
@NullVoxPopuli don't kill me, but can we put the new lookup chain in a separate PR and just add the `namespace` code in this PR?